Description: systemd: Fix ntpviz dependencies
 The ntpviz services should not Requires=ntpd.service.  That has the effect of
 starting ntpd.service (if it was stopped) whenever the timer fires.
Forwarded: no
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-09
--- a/etc/ntpviz-daily.service
+++ b/etc/ntpviz-daily.service
@@ -1,7 +1,6 @@
 [Unit]
 Documentation=man:ntpviz(1)
 Description=Graph daily information for NTP
-Requires=ntpd.service
 
 [Service]
 Type=simple
--- a/etc/ntpviz-weekly.service
+++ b/etc/ntpviz-weekly.service
@@ -1,7 +1,6 @@
 [Unit]
 Documentation=man:ntpviz(1)
 Description=Graph weekly information for NTP
-Requires=ntpd.service
 
 [Service]
 Type=simple
